src/rust/bitbox02/src/lib.rs

```diff
@@ -227,3 +230,19 @@ pub fn sdcard_inserted() -> bool {
     let data = crate::testing::DATA.0.borrow();
     data.sdcard_inserted.as_ref().unwrap()()
 }
+
+pub fn format_datetime(timestamp: u32, timezone_offset: i32, date_only: bool) -> String {
+    let mut out = [0u8; 100];
```

Comment:
I think 50 would've been more than enough?

Reply:
eh, i just copied what we had in C :P https://github.com/digitalbitbox/bitbox02-firmware/blob/d10f9f79e6c71b0da2e3ffab308e27fb3d862a10/src/workflow/confirm_time.c#L25